ansible/roles/metricbeat/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Don't we have already metric beats in https://github.com/dashpay/dash-network-deploy/pull/617. How do you replace Prometheus here?

Reply:
We have a role named metrics in ansible which is currently monitoring using prometheus.  Metricbeat role will be applied on the logs instance where the elastic_stack role is applied and it will target the hosts which are currently being monitored using prometheus and since this role in applied on the same logs instances , it will send  the data to the elasticsearch running on that instance. (configured using the localhost and using network name)

Reply:
We already have metric beats on the nodes where we are running tenderdash. Why not just configure them to collect data from tenderdash and send to elastic?